Userland/Applications/Maps/MapsMapWidget.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2023, Bastiaan van der Plaat <bastiaan.v.d.plaat@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "MapWidget.h"
+
+class MapsMapWidget final : public MapWidget {
```

Comment:
`MapsMapWidget` is a pretty opaque name; what about `UsersMapWidget`?

Reply:
:laughing: Oh no.

This is my fault. Originally they picked UsersMapWidget but to me, that's limiting when we're going to display other things in this widget.

Reply:
Hah I should have checked your review as well, but seeing as this is very specialized a `Users` designation doesn't seem to be that far off. If the downloading/parsing part is abstracted out of this class, it could maybe be a MarkersMapWidget?

Reply:
Showing markers is built into MapWidget too. I did previously suggest not using a subclass:
> Loading the users data in main() and just telling MapWidget to add the markers, instead of making it responsible for loading and parsing. (The grey-marker bitmap would also be loaded there.)

(Part of me wonders about using GUI::Model for the backing data but that's probably not super useful?)

Reply:
Oke, I renamed the class name back to `UsersMapWidget.cpp` because it is a more clear name. I understand that it is unwanted that we extend the map a lot of times to inheritance to add map items. But I also don't want to blow up `main.cpp` that is just the window chrome that holds the main Map Widget in my opinion.